Translatron.cs

```diff
@@ -0,0 +1,118 @@
+﻿using KRPC.Service.Attributes;
+using System.Reflection;
+
+namespace KRPC.MechJeb
+{
+    /// <summary>
+    /// The Translatron module controls the vessel's throttle/velocity.
+    /// </summary>
+    [KRPCClass(Service = "MechJeb")]
+    public class Translatron : DisplayModule
```

Comment:
There is a typo: it should be `Translation`, not `Translatron`. The file name also needs changing. Can you do that, please?